packages/components/statistic/src/index.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <div :class="[ns.b()]">
+    <div :class="ns.e('head')">
+      <slot name="title">
+        <span :class="ns.e('title')">
```

Comment:
It seems written backwards.
```js
<span :class="ns.e('title')">
  <slot name="title">
```

Reply:
At that time, the layout was considered to be controlled by the head, and the internal styles would not be preserved if they were customized

Reply:
At that time, the layout was considered to be controlled by the head, and the internal styles would not be preserved if they were customized

Reply:
What do you think

Reply:
I consider `ns.e('title')` is a default class. Users can modify the styles or define something new to cover it. So do Other components like `Input` .